applications/projects/Regression/ExternalProjectConfig.cmake.in

```diff
@@ -2,8 +2,8 @@ cmake_minimum_required(VERSION 2.8.2)
 
 include(ExternalProject)
 ExternalProject_Add(Regression
-    GIT_REPOSITORY https://github.com/sofa-framework/regression
-    GIT_TAG db26140ba6523645627a98c90ed751df4f78cd1c
+    GIT_REPOSITORY https://github.com/fredroy/regression
+    GIT_TAG origin/follow_sofa_pr1834
```

Comment:
You can push your changes directly on https://github.com/sofa-framework/Regression and update GIT_TAG only :+1:

Reply:
Ok i will push directly the branch in the sofa-framework repo from now on then